### ISSUE TITLE
Suggested Feedback on Example Notebook

### ISSUE THREAD

**Issue by RylanSchaeffer** [nicholas-roy/psytrack#9]
I'd like to give some suggested feedback on the example notebook:

https://github.com/nicholas-roy/psytrack/blob/master/psytrack/examples/ExampleNotebook.ipynb

1. Almost immediately, the tutorial uses `generateSim()` and `recoverSim()` functions, but there's no comments detailing what these functions are doing or what their inputs/outputs are. This could be remedied by printing the types, shapes, etc. of the input and output objects.

2. Looking at the code for `generateSim()`, the documentation states that the function returns either:
```
        save_path | (if savePath) : str, the name of the folder+file where
            simulation data was saved in the local directory
        save_dict | (if no SavePath) : dict, contains all relevant info
            from the simulation 
```

But unless I read the paper and the code, I have no idea what "all relevant info from the simulation" is.

**Comment by RylanSchaeffer** [comment]
3. I'd like the example notebook to specify how long `hyperOpt()`  should take to run. Right now I've been waiting 15 minutes with relatively small inputs (number of trials = 6000, two weights, each with 1 column) and I don't know if this to be expected. If it isn't, hopefully the example notebook would clarify how to debug the source of the slowness.

**Comment by RylanSchaeffer** [comment]
4. It'd be great if `makeWeightPlot` returned the plot object.

**Comment by nicholas-roy** [comment]
Thank you for all the suggestions:

1+2) The notebook has been updated to make clear that the short bit with simulated data is a _demonstration_, and that the main tutorial is below!

3) We found the bug that was causing your optimization to slow offline. This bug should not be an issue in the new release, version 1.2.0

4) The plotting functions have been given an overhaul — check out the refreshed example notebook for an example, should make things a lot easier!

Thanks again!
